igvm/hypervisor.py

```diff
@@ -57,12 +57,27 @@ def __init__(self, dataset_obj):
         # We cannot store these in the VM object due to migrations.
         self._mount_path = {}
 
-    def vm_disk_path(self, name):
-        return '/dev/{}/{}'.format(VG_NAME, name)
+    def vm_lv_get(self, vm):
+        """Get a VMs logical volume information"""
```

Comment:
"Gets VM's ..." or "Get ... of a VM".

Reply:
Certainly not "Gets VM's ..." :P
I'll make it "Get logical volume information of a VM"